src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
Properties `justify-content` and `align-items` should be applied to flex or grid containers. Since `body` is not a flex or grid container itself, these properties have no effect here. Consider applying a flex container around the content that needs to be centered if necessary.